### ISSUE TITLE
Unable to change playback speed on dev build

### ISSUE THREAD

**Issue by Thomasedv** [clsid2/mpc-hc#1083]
Can provide sample video if required. Using latest dev build (MPC-HC.1.9.11.14.x64.exe)

Pressing the buttons to change playbacks speed does nothing, and I have tested it on multiple videos. Pressing the buttons just causes a split second stutter before it continues as if nothing changed. No speed notification (eg. 0.5x) shows when pressing the buttons either. 

Issue is not present on latest release. (1.9.11)

I'm on Windows 10, running latest updates, MadVR renderer, RTX 3090 GPU. Don't know if any of this information is relevant, or if there is something else I should provide.

**Comment by Thomasedv** [comment]
Reinstalled the latest dev version and it worked...

**Comment by Thomasedv** [comment]
Turns out all my shadowplay recordings have stopped working. I could have sworn it wasn't like this before, but all my recordings simply can't change playback speed, regardless of mpc-hc version I have installed. I don't know if this is an issue with mpc-hc, but I doubt it. I could share a clip if you want to investigate.

**Comment by clsid2** [comment]
There can be two reasons why speed change doesn't work:
1) One of the filters in the playback chain doesn't support speed change.
2) The audio has a high sample rate. The default audio renderer has a maximum it supports, which limits how much playback can be sped up.

Try enabling the internal audio renderer. That resamples the audio so it doesn't have speed up limitations.
Options > Playback > output

**Comment by Thomasedv** [comment]
Thank you! That solved the issue. I made some interesting discovery, which may or may not be interesting to you, but i thought i'd mention it. First off all, my output was set to my optical output device, which rendered me completely unable to change playback speed.

As per your suggestion, I set it to internal renderer, which worked. Only problem is that the sound skips like a broken record when slowed down, absolutely horrible to listen to. Then, moving to just using "System default" I was allowed to use slower playback speed, and audio got properly distorted instead of repeating the same segments over and over again. Which is the behavior I'm used to from before. I suppose I must have changed it to the optical output at some point and not realized the consequences of that. 

Again, Thanks for the help!
